cglims/cli/commands.py

```diff
@@ -119,7 +119,8 @@ def get(context, condense, project, external, identifier, field, all_samples):
         values['is_human'] = clinical_sample.apptag.is_human
         values['is_production'] = (False if values['customer'] == 'cust000'
                                    else True)
-        values['pipeline'] = clinical_sample.pipeline
+        pipeline = clinical_sample.pipeline
+        values['pipeline'] = pipeline if pipeline else 'NA'
```

Comment:
When it gets printed, one wants to see an `NA`, not a `null` or `None`. I tested it out with `None` as well, and it strangely resulted in `null` being printed. I presume this is the `click.echo` converting `None` to `null`?